lib/string_tools/core_ext/string.rb

```diff
@@ -154,10 +154,11 @@ def remove_colors
     end
   end
 
+  WIN_1251_ENCODING = 'windows-1251'.freeze
   # shorthand
   def detect_encoding
     e = ::CharDet.detect(self)["encoding"]
-    e = 'windows-1251' if StringTools.cp1251_compatible_encodings.include?(e)
+    e =  WIN_1251_ENCODING if StringTools::CharDet::CP1251_COMPATIBLE_ENCODINGS.include?(e)
```

Comment:
Layout/SpaceAroundOperators: Operator = should be surrounded by a single space.<br>Layout/ExtraSpacing: Unnecessary spacing detected.